build.config

```diff
@@ -1,5 +1,5 @@
 # Path to AIR SDK
-air.sdk = {AIR_SDK_PATH}
+air.sdk = /Applications/Adobe/air20
```

Comment:
this change should only be a local change.

this isn't your fault, but this file should be `build.config.dist`, and then `build.config` should be in the `.gitignore`